src/pypromice/postprocess/args.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('--dev',
+    action='store_true',
+    help='If included (True), run in dev mode. Useful for repeated runs of script between transmissions.')
+
+parser.add_argument('--positions',
+    action='store_true',
+    help='If included (True), make a positions dict and output a positions.csv file.')
+
+parser.add_argument('--l3-filepath',
+    default='../../../../aws-l3/tx/*/*_hour.csv',
```

Comment:
I think pre-defined file paths should be taken out of this. If this `csv2bufr` processing is moved to a command line function then this will have to change anyway.